liiweb/templates/peachjam/_footer.html

```diff
@@ -13,6 +13,11 @@ <h2 class="mb-2">{% trans 'Subscribe' %}</h2>
     <div class="col-12 col-md-6 mb-2">
       {% block newsletter-form-code %}{% endblock %}
     </div>
+    {% if PEACHJAM_SETTINGS.facebook_link or PEACHJAM_SETTINGS.twitter_link %}
```

Comment:
If you move this into the above `div`, then you don't need to worry about cols and rows and stuff.